ThingIFSDK/ThingIFSDK/ThingIFAPI+GetState.swift

```diff
@@ -9,39 +9,50 @@
 import Foundation
 
 extension ThingIFAPI {
-    func _getState(
-        _ completionHandler: @escaping (Dictionary<String, Any>?,  ThingIFError?)-> Void
-        ){
-        fatalError("TODO: implement me.")
-        /*
-            guard let target = self.target else {
-                completionHandler(nil, ThingIFError.targetNotAvailable)
-                return
-            }
+    func _getTargetStates(
+        _ completionHandler: @escaping ([String : [String : Any]]?,  ThingIFError?)-> Void) {
+
+        guard let target = self.target else {
+            completionHandler(nil, ThingIFError.targetNotAvailable)
+            return
+        }
+
+        self.operationQueue.addHttpRequestOperation(
+            .GET,
+            url: "\(baseURL)/thing-if/apps/\(appID)/targets/\(target.typedID.toString())/states",
+            requestHeader:
+            self.defaultHeader + ["Content-Type" : "application/json" ],
```

Comment:
@naoki-akimoto Is `Content-Type` required? I checked the [specification](https://docs.google.com/document/edit?hgd=1&id=1TLZvjKvwSQ_C66XcC7haef5-PtO639y2fjhQIlOBCYE#heading=h.a82wsgp1p2a2) and I did not found `Content-Type`. Could you double check it? 

If `Content-Type` is not required, Please remove this.